Ginger/GingerCoreCommon/Repository/RepositoryFolder.cs

```diff
@@ -432,15 +432,18 @@ private void FileWatcher_Changed(object sender, FileSystemEventArgs e)
         private void HandleFileChange(FileSystemEventArgs e)
         {            
             RepositoryItemBase item = null;
-
+            RepositoryFolderBase folderItem = null;
             switch (e.ChangeType)
             {
                 case WatcherChangeTypes.Changed:
                     WaitforFileIsReadable(e.FullPath);
                     // reLoad the object to mem updating fields
                     item = GetItemFromCacheByFileName(e.FullPath);
+                    folderItem = GetSubFolder(Path.GetDirectoryName(e.FullPath));
+                    folderItem.RefreshSourceControlStatus();
+                    folderItem.OnPropertyChanged(nameof(RepositoryItemBase.SourceControlStatus));
                     NewRepositorySerializer.ReloadObjectFromFile(item);
```

Comment:
folderItem.RefreshSourceControlStatus();
folderItem.OnPropertyChanged(nameof(RepositoryItemBase.SourceControlStatus));
above 2 code line is doing the same thing, no?